controllers/machinehealthcheck_controller_test.go

```diff
@@ -46,6 +46,71 @@ import (
 
 const defaultNamespaceName = "default"
 
+func newMachine(mhc *clusterv1.MachineHealthCheck, cluster *clusterv1.Cluster) clusterv1.Machine {
+	return clusterv1.Machine{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: clusterv1.GroupVersion.String(),
+			Kind:       "Machine",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "test-mhc-machine-",
+			Namespace:    mhc.Namespace,
+			Labels:       mhc.Spec.Selector.MatchLabels,
+		},
+		Spec: clusterv1.MachineSpec{
+			ClusterName: cluster.Name,
+			Bootstrap: clusterv1.Bootstrap{
+				Data:           pointer.StringPtr("data"),
```

Comment:
This field is deprecated and replaced by DataSecretName. It should be safe (and better) to remove this line, I think.